src/Classes/ModuleCreator.php

```diff
@@ -119,7 +145,7 @@ public function createModuleInfoJsonFile($vendorName, $moduleName)
             ],
 
             'modifiedCompatibility' => [
-                '2.0.4.2'
+                $this->getCurrentModifiedVersion() . ',',
```

Comment:
You can use `ShopInfo::getModifiedVersion();`

Reply:
I hope this isn't too frustrating since you wrote so much code. However, if you think your function is better at getting the modified version, you can improve App:getModifedVersion() and use the improved App:getModifedVersion().

Reply:
> You can use ShopInfo::getModifiedVersion();

Neat, thanks!

> I hope this isn't too frustrating since you wrote so much code.

Not at all!

> However, if you think your function is better at getting the modified version, you can improve ShopInfo:getModifedVersion() and use the improved ShopInfo:getModifedVersion()

It's funny because [I helped write that method](https://github.com/RobinTheHood/ModifiedModuleLoaderClient/blame/master/src/Classes/ShopInfo.php#L38-L69) 😄.